### PR TITLE
Import component in component

### PR DESCRIPTION
**Pull request by helloimela**
- Add helpers.js to import components as a module in project
- Add function to automatically import sub components that exist inside component's template
- helpers.js is available in the dist folder

How to use in project:
Import `defineCustomElements` function :
`import { defineCustomElements } from 'node_modules/corporate-ui/dist/collection/helpers';`
Define custom elements needed in the project : 
`defineCustomElements(['c-header','c-footer','c-theme']);`

Fixing issue #32

### CODE REVIEW

**Comment by helloimela** [comment]
Should be compiled with webpack, separated issue created #45

File: src/helpers.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { defineCustomElement as defineCE } from '../esm/es5/corporate-ui.core'
```
Comment (awinny) [inline src/helpers.js]: As we are now exporting "defineCustomElements", the as syntax is no longer needed

**Comment by awinny** [comment]
Stange that the prettifier did not run with same settings as mine.
Can be seen in stencil.config.ts
Seems to be the same when it comes to the tslint, for example: "if (comp[0] === name) val = comp" should have been rendered with curly braces.

**Comment by awinny** [comment]
We should also have a demonstration for this I think.
As a start we can use it in the storybook setup.

File: src/helpers.js (hunk 1 above)
Comment (helloimela) [reply src/helpers.js]: That's correct...will change it and add a demo in Storybook

**Pull request merged**